tableauserverclient/server/endpoint/schedules_endpoint.py

```diff
@@ -68,12 +68,12 @@ def create(self, schedule_item):
         return new_schedule
 
     @api(version="2.8")
-    def add_to_schedule(self, schedule_id, workbook=None, datasource=None):
+    def add_to_schedule(self, schedule_id, workbook=None, datasource=None, schedule_type='extractRefresh'):
```

Comment:
All of these schedule_type/task_type arguments should use enums rather than raw strings.

Reply:
Added TaskItem.Type. 

Here is an inconsistency. When calling [get_extract_refresh_tasks](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref.htm#get_extract_refresh_tasks), the word for extract refresh is `extractRefreshes". However, the word in the request body for [add_workbook_to_schedule ](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref.htm#add_workbook_to_schedule) is extractRefresh. I added a normalize function in tasks_endpoint.py to deal with this inconsistency.